2021-11-17 13:23:09,814 INFO    MainThread:13492 [wandb_setup.py:_flush():71] setting env: {}
2021-11-17 13:23:09,815 INFO    MainThread:13492 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-17 13:23:09,815 INFO    MainThread:13492 [wandb_init.py:_log_setup():357] Logging user logs to /mnt/c/Users/wanga/Amian/persional/ntnu_homework/IR/HW4/wandb/run-20211117_132309-t474s9co/logs/debug.log
2021-11-17 13:23:09,815 INFO    MainThread:13492 [wandb_init.py:_log_setup():358] Logging internal logs to /mnt/c/Users/wanga/Amian/persional/ntnu_homework/IR/HW4/wandb/run-20211117_132309-t474s9co/logs/debug-internal.log
2021-11-17 13:23:09,816 INFO    MainThread:13492 [wandb_init.py:init():390] calling init triggers
2021-11-17 13:23:09,816 INFO    MainThread:13492 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-17 13:23:09,822 INFO    MainThread:13492 [wandb_init.py:init():435] starting backend
2021-11-17 13:23:09,822 INFO    MainThread:13492 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-17 13:23:09,830 INFO    MainThread:13492 [backend.py:ensure_launched():199] starting backend process...
2021-11-17 13:23:09,832 INFO    MainThread:13492 [backend.py:ensure_launched():205] started backend process with pid: 13524
2021-11-17 13:23:09,833 INFO    MainThread:13492 [wandb_init.py:init():444] backend started and connected
2021-11-17 13:23:09,834 INFO    MainThread:13492 [wandb_init.py:init():503] updated telemetry
2021-11-17 13:23:09,845 INFO    MainThread:13492 [wandb_init.py:init():533] communicating current version
2021-11-17 13:23:10,325 INFO    MainThread:13492 [wandb_init.py:init():538] got version response 
2021-11-17 13:23:10,326 INFO    MainThread:13492 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-17 13:23:10,742 INFO    MainThread:13492 [wandb_init.py:init():576] starting run threads in backend
2021-11-17 13:23:12,326 INFO    MainThread:13492 [wandb_run.py:_console_start():1693] atexit reg
2021-11-17 13:23:12,328 INFO    MainThread:13492 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-17 13:23:12,329 INFO    MainThread:13492 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-17 13:23:12,331 INFO    MainThread:13492 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-17 13:23:12,331 INFO    MainThread:13492 [wandb_init.py:init():603] run started, returning control to user process
2021-11-17 13:23:12,332 INFO    MainThread:13492 [wandb_run.py:_config_callback():911] config_cb None None {'docs_path': '../datasets/q_100_d_10000/docs', 'queries_path': '../datasets/q_100_d_10000/queries', 'epochs': 100}
2021-11-17 13:23:37,965 INFO    MainThread:13492 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-17 13:23:37,966 INFO    MainThread:13492 [wandb_run.py:_restore():1635] restore
2021-11-17 13:23:40,681 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 590
}

2021-11-17 13:23:41,011 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 590
}

2021-11-17 13:23:41,958 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 6308
}

2021-11-17 13:23:42,094 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 590
  total_bytes: 6335
}

2021-11-17 13:23:42,198 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:42,301 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:42,405 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:42,508 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:42,611 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:42,719 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:43,321 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}

2021-11-17 13:23:43,798 INFO    MainThread:13492 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6335
  total_bytes: 6335
}
local_info {
}

2021-11-17 13:23:44,857 INFO    MainThread:13492 [wandb_run.py:_append_files():2061] logging synced files
